Source/Provers/SMTLib/SMTLibProcessTheoremProver.cs

```diff
@@ -1951,7 +1951,20 @@ public override int GetRCount()
     {
       SendThisVC("(get-info :rlimit)");
       var resp = Process.GetProverResponse();
-      return int.Parse(resp[0].Name);
+      try
+      {
+        return int.Parse(resp[0].Name);
+      }
+      catch
+      {
+        // If anything goes wrong with parsing the response from the solver,
+        // it's better to be able to continue, even with uninformative data.
+        lock (proverWarnings)
+        {
+          proverWarnings.Add("Failed to parse resource count from solver.");
```

Comment:
As per below, let's include the response in the warning

Reply:
Agreed.